Chinatown expansion!

xxxnoscopexx opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Added a casino, restaurant, and underground base to chinatown.

## Why It's Good For The Game
adds some new places to roleplay lets the triads control  a bit more of their home and gives humans another place to eat and more food.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Added new casino! gamble your heart away!
add: Added new restaurant for the triad with a extra treat in the back!(guns shop) 
add: Added new unground triad with meth lab and armory.
del: removed old chinatown gunshop. 
del: removed weird hobo sewer side area converted into sewer entrance of triad base. 
del: removed Ak and Uzi from weed farm above laundromat and moved it to unground armory.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

faaaay commented:
Would it be possible to add screenshots of the additions?

spartanyanni commented:
let's go gambling

spartanyanni commented:
Just a heads up, some of the keys scattered through the Laundromat currently (Ever since the second story expansion) don't work with some of the doors. Please, respectfully, double check them against any doors you might be adding/changing.

xxxnoscopexx commented:
Old and New chinatown 
![changes](https://github.com/user-attachments/assets/f0e8adff-621c-484d-bf54-de13f256abb2)
new underground
![image](https://github.com/user-attachments/assets/840fdb66-a2bb-485e-a64f-ffae5df5fa6f)
new underground location
![image_2025-01-05_074346711](https://github.com/user-attachments/assets/058c27b7-56cf-4329-989e-311994c0f7fe)

xxxnoscopexx commented:
added new chef keys that are a copy of the laundromat and made laundromat keys and made them both work specifically on all the new front and side doors for the casino and restaurant in case triads wanna hire someone to manage them without giving them base access and added another pair of triad keys in the base to give or if someone needs a spare.

Metekillot commented:
Should be fine to merge once I spin this up to verify door sanity and anything else

xxxnoscopexx commented:
new sorta walk through as I was testing doors
![image](https://github.com/user-attachments/assets/136d5795-bd8f-4014-b12b-7547dd35799c)
![image](https://github.com/user-attachments/assets/0a182e0c-3e72-42e5-a4d5-f7aac441632a)
![image](https://github.com/user-attachments/assets/d51cadc8-5a0b-4a58-be6a-253579e46b1c)
![image](https://github.com/user-attachments/assets/a0fc979e-89db-4691-8291-01ef03be3a8f)

cheapcents reviewed (changes requested):
![image](https://github.com/user-attachments/assets/6328d5ab-10b1-47e8-8614-9682e555078b)
offset door
![image](https://github.com/user-attachments/assets/e90ac16d-69f8-4753-bb12-60f77cd721f5)
what exactly is this checkpoint supposed to be...?

sorry for the long wait, by the way

xxxnoscopexx commented:
checkpoint was just for safety since its dangerous in the sewers and im fixing that and the lights right now it slipped my mind

xxxnoscopexx commented:
![image](https://github.com/user-attachments/assets/3c4769c7-b86c-4a18-b3d1-80e30d42f3c9)

xxxnoscopexx commented:
![image](https://github.com/user-attachments/assets/16f5f2a2-6717-485c-a30a-a933f9b53112)
various stuff in the lockers of the casino

cheapcents reviewed (approved):
All seems to be in order, good work
